pkg/controller/directvolumemigration/stunnel.go

```diff
@@ -60,13 +54,13 @@ metadata:
     purpose: stunnel
 data:
   stunnel.conf: |
-    foreground = yes
     pid =
     sslVersion = TLSv1.2
     client = yes
     syslog = no
+    output = /dev/stdout
     [rsync]
-    accept = {{ .StunnelPort }}
+    accept = localhost:2222
```

Comment:
any reason for not using `localhost:{{ .StunnelPort }}` ?

Reply:
Done, updated the PR, had missed it, thanks :)